<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gerenciador de transações</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Icons/crosslifeIcon.jpg</normaloff>Icons/crosslifeIcon.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(33, 37, 43);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1421</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 25 12pt &quot;Segoe UI Light&quot;;
color: white;

background-color: rgb(33, 37, 43);
background-color: rgb(102, 138, 172);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="Line" name="line_3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>526</y>
            <width>84</width>
            <height>16</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_8">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>429</y>
            <width>60</width>
            <height>60</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::DefaultContextMenu</enum>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
  padding: 15px 25px;
  font-size: 24px;
  text-align: center;
  color: #fff;
  	border: 2px solid rgb(52, 59, 72);
	border-radius: 15px;	
	background-color: rgb(52, 59, 72);
} QPushButton:hover {
	background-color: rgb(57, 65, 80);
	border: 2px solid rgb(61, 70, 86);
}QPushButton:pressed {	
	background-color: rgb(35, 40, 49);
	border: 2px solid rgb(43, 50, 61);
}



</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../../Downloads/icons8-reinicialização-50.png</normaloff>../../../Downloads/icons8-reinicialização-50.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>50</width>
            <height>35</height>
           </size>
          </property>
         </widget>
         <widget class="QDateTimeEdit" name="dateTimeEdit">
          <property name="geometry">
           <rect>
            <x>-600</x>
            <y>20</y>
            <width>0</width>
            <height>0</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: transparent;
background-color: transparent;</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_7">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>138</y>
            <width>60</width>
            <height>60</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Botão pesquisar. A pesquisa é feita de acordo com o tipo de pesquisa selecionado acima.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="whatsThis">
           <string/>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
  padding: 15px 25px;
  font-size: 24px;
  text-align: center;
  color: #fff;
  	border: 2px solid rgb(52, 59, 72);
	border-radius: 15px;	
	background-color: rgb(52, 59, 72);
} QPushButton:hover {
	background-color: rgb(57, 65, 80);
	border: 2px solid rgb(61, 70, 86);
}QPushButton:pressed {	
	background-color: rgb(35, 40, 49);
	border: 2px solid rgb(43, 50, 61);
}



</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../../Downloads/icons8-pesquisar-50 (1).png</normaloff>../../../Downloads/icons8-pesquisar-50 (1).png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
         </widget>
         <widget class="QPushButton" name="buttonSelectDate">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>235</y>
            <width>60</width>
            <height>60</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
  padding: 15px 25px;
  font-size: 24px;
  text-align: center;
  color: #fff;
  	border: 2px solid rgb(52, 59, 72);
	border-radius: 15px;	
	background-color: rgb(52, 59, 72);
} QPushButton:hover {
	background-color: rgb(57, 65, 80);
	border: 2px solid rgb(61, 70, 86);
}QPushButton:pressed {	
	background-color: rgb(35, 40, 49);
	border: 2px solid rgb(43, 50, 61);
}



</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../../Downloads/icons8-calendário-50.png</normaloff>../../../Downloads/icons8-calendário-50.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_6">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>332</y>
            <width>60</width>
            <height>60</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::DefaultContextMenu</enum>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
  padding: 15px 25px;
  font-size: 24px;
  text-align: center;
  color: #fff;
  	border: 2px solid rgb(52, 59, 72);
	border-radius: 15px;	
	background-color: rgb(52, 59, 72);
} QPushButton:hover {
	background-color: rgb(57, 65, 80);
	border: 2px solid rgb(61, 70, 86);
}QPushButton:pressed {	
	background-color: rgb(35, 40, 49);
	border: 2px solid rgb(43, 50, 61);
}



</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../../Downloads/icons8-pdf-50 (1).png</normaloff>../../../Downloads/icons8-pdf-50 (1).png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>50</width>
            <height>35</height>
           </size>
          </property>
         </widget>
         <zorder>pushButton_7</zorder>
         <zorder>buttonSelectDate</zorder>
         <zorder>pushButton_6</zorder>
         <zorder>dateTimeEdit</zorder>
         <zorder>line_3</zorder>
         <zorder>pushButton_8</zorder>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(33, 37, 43);</string>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="styleSheet">
          <string notr="true">QFrame{background-color: rgb(75, 83, 98);}


QScrollBar:horizontal {
    border: none;
    background: rgb(52, 59, 72);
    height: 8px;
    margin: 0px 21px 0 21px;
	border-radius: 0px;
}
QScrollBar::handle:horizontal {
    background: rgb(189, 147, 249);
    min-width: 25px;
	border-radius: 4px
}
QScrollBar::add-line:horizontal {
    border: none;
    background: rgb(55, 63, 77);
    width: 20px;
	border-top-right-radius: 4px;
    border-bottom-right-radius: 4px;
    subcontrol-position: right;
    subcontrol-origin: margin;
}
QScrollBar::sub-line:horizontal {
    border: none;
    background: rgb(55, 63, 77);
    width: 20px;
	border-top-left-radius: 4px;
    border-bottom-left-radius: 4px;
    subcontrol-position: left;
    subcontrol-origin: margin;
}
QScrollBar::up-arrow:horizontal, QScrollBar::down-arrow:horizontal
{
     background: none;
}
QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal
{
     background: none;
}
 QScrollBar:vertical {
	border: none;
    background: rgb(52, 59, 72);
    width: 8px;
    margin: 21px 0 21px 0;
	border-radius: 0px;
 }
 QScrollBar::handle:vertical {	
	background: rgb(189, 147, 249);
    min-height: 25px;
	border-radius: 4px
 }
 QScrollBar::add-line:vertical {
     border: none;
    background: rgb(55, 63, 77);
     height: 20px;
	border-bottom-left-radius: 4px;
    border-bottom-right-radius: 4px;
     subcontrol-position: bottom;
     subcontrol-origin: margin;
 }
 QScrollBar::sub-line:vertical {
	border: none;
    background: rgb(55, 63, 77);
     height: 20px;
	border-top-left-radius: 4px;
    border-top-right-radius: 4px;
     subcontrol-position: top;
     subcontrol-origin: margin;
 }
 QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
     background: none;
 }

 QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
     background: none;
 }


QRadioButton::indicator {
    border: 3px solid rgb(52, 59, 72);
	width: 15px;
	height: 15px;
	border-radius: 10px;
    background: rgb(44, 49, 60);
}
QRadioButton::indicator:hover {
    border: 3px solid rgb(58, 66, 81);
}
QRadioButton::indicator:checked {
    background: 3px solid rgb(94, 106, 130);
	border: 3px solid rgb(52, 59, 72);	
}

/* /////////////////////////////////////////////////////////////////////////////////////////////////
ComboBox */
QComboBox{
	background-color: rgb(27, 29, 35);
	border-radius: 5px;
	border: 2px solid rgb(33, 37, 43);
	padding: 5px;
	padding-left: 10px;
}
QComboBox:hover{
	border: 2px solid rgb(64, 71, 88);
}
QComboBox::drop-down {
	subcontrol-origin: padding;
	subcontrol-position: top right;
	width: 25px; 
	border-left-width: 3px;
	border-left-color: rgba(39, 44, 54, 150);
	border-left-style: solid;
	border-top-right-radius: 3px;
	border-bottom-right-radius: 3px;	
	background-image: url(:/icons/images/icons/cil-arrow-bottom.png);
	background-position: center;
	background-repeat: no-reperat;
 }
QComboBox QAbstractItemView {
	color: rgb(255, 121, 198);	
	background-color: rgb(33, 37, 43);
	padding: 10px;
	selection-background-color: rgb(39, 44, 54);
}

</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QTableWidget" name="tableWidget">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QTableWidget{
color: rgb(255, 255, 255);
}
QHeaderView::section{
font: 12pt &quot;MS Shell Dlg 2&quot;;
 background-color:grey
border-stye: none;


}</string>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::ExtendedSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>110</number>
            </attribute>
            <attribute name="horizontalHeaderMinimumSectionSize">
             <number>110</number>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
      <zorder>frame_4</zorder>
      <zorder>frame_3</zorder>
      <zorder>line_2</zorder>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(33, 37, 43);</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="maximumSize">
       <size>
        <width>9898989</width>
        <height>120</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(33, 37, 43);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QFrame" name="frame_5">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(33, 37, 43);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="pushButton_5">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>80</height>
             </size>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Botão voltar. Volta ao modulo inicial.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
  padding: 15px 25px;
  font-size: 24px;
  text-align: center;
	color: #FA5252;
  	border: 2px solid rgb(52, 59, 72);
	border-radius: 5px;	
	background-color: rgb(52, 59, 72);
} QPushButton:hover {
	background-color: rgb(57, 65, 80);
	border: 2px solid rgb(61, 70, 86);
}QPushButton:pressed {	
	background-color: rgb(35, 40, 49);
	border: 2px solid rgb(43, 50, 61);
}



</string>
            </property>
            <property name="text">
             <string>Fechar</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../../../Downloads/icons8-excluir-50 (6).png</normaloff>../../../Downloads/icons8-excluir-50 (6).png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_6">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(33, 37, 43);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Botão inserir. Abre uma nova janela para a inserção de registros.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
  padding: 15px 25px;
  font-size: 24px;
  text-align: center;
  color: #fff;
  	border: 2px solid rgb(52, 59, 72);
	border-radius: 5px;	
	background-color: rgb(52, 59, 72);
} QPushButton:hover {
	background-color: rgb(57, 65, 80);
	border: 2px solid rgb(61, 70, 86);
}QPushButton:pressed {	
	background-color: rgb(35, 40, 49);
	border: 2px solid rgb(43, 50, 61);
}



</string>
            </property>
            <property name="text">
             <string>Inserir</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../../../Downloads/icons8-inserir-50 (1).png</normaloff>../../../Downloads/icons8-inserir-50 (1).png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Botão alterar. Abre uma nova janela para alteração de registros.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
  padding: 15px 25px;
  font-size: 24px;
  text-align: center;
  color: #fff;
  	border: 2px solid rgb(52, 59, 72);
	border-radius: 5px;	
	background-color: rgb(52, 59, 72);
} QPushButton:hover {
	background-color: rgb(57, 65, 80);
	border: 2px solid rgb(61, 70, 86);
}QPushButton:pressed {	
	background-color: rgb(35, 40, 49);
	border: 2px solid rgb(43, 50, 61);
}



</string>
            </property>
            <property name="text">
             <string>Alterar</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../../../Downloads/icons8-17372-0-73111-repetição-direita-43-setas-64.png</normaloff>../../../Downloads/icons8-17372-0-73111-repetição-direita-43-setas-64.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>60</width>
              <height>25</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_4">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Botão excluir. Abre uma nova janela para e exclusão de registros.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
  padding: 15px 25px;
  font-size: 24px;
  text-align: center;
  color: #fff;
  	border: 2px solid rgb(52, 59, 72);
	border-radius: 5px;	
	background-color: rgb(52, 59, 72);
} QPushButton:hover {
	background-color: rgb(57, 65, 80);
	border: 2px solid rgb(61, 70, 86);
}QPushButton:pressed {	
	background-color: rgb(35, 40, 49);
	border: 2px solid rgb(43, 50, 61);
}



</string>
            </property>
            <property name="text">
             <string>Excluir</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../../../Downloads/icons8-delete-64.png</normaloff>../../../Downloads/icons8-delete-64.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>50</width>
              <height>25</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
